artifacts/CAMARA_common.yaml

```diff
@@ -1,326 +1,340 @@
 openapi: 3.0.3
 info:
- title: Camara common data types
- description: Common data types for Camara APIs
- contact:
-  email: sp-com@lists.camaraproject.org
- license:
-  name: Apache 2.0
-  url: https://www.apache.org/licenses/LICENSE-2.0.html
- version: 0.4.0
+  title: Camara common data types
+  description: Common data types for Camara APIs
+  contact:
+    email: sp-com@lists.camaraproject.org
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.4.0

```

Comment:
Is it needed to move to 0.5.0?

Reply:
I'm not sure about the management of the version for this document. 
My 2 cents: we should try to keep aligned this document version with the guideline version? so in this cas it should be 0.4.0-wip?

Reply:
@bigludo7 Do you mean here the repo-release version? I could not find any version specified in the guidelines doc.

Reply:
I was thinking of the version in this document but I've probably mixing myself with the branch version :)

Reply:
:) Then, we could simply have wip? 
(common.yaml version was already 0.4 in comms release 0.3, so we would need to increment the version for the upcoming rel(0.4) to 0.5)

Reply:
Yes just wip will be good enough !